mmcv/runner/hooks/logger/tensorboard.py

```diff
@@ -1,15 +1,20 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import os.path as osp
+from functools import partial
 
-from mmcv.utils import TORCH_VERSION, digit_version
+import torch
+
+from ....parallel import DataContainer
+from ....parallel.utils import is_module_wrapper
+from ....utils import TORCH_VERSION, digit_version
```

Comment:
Relative package path with more than two `.` is weird. Why don't use absolte path?

Reply:
I will fix it.